_clients/opensearch-py-ml.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Opensearch-py-ml
+nav_order: 11
+---
+
+`opensearch-py-ml` is a Python client that provides a suite of data analytics and natural language processing (NLP) support tools for OpenSearch. It provides data analysts with the following functionality:
+
+- Call OpenSearch indexes and manipulate using [eland.DataFrame](https://eland.readthedocs.io/en/v8.3.0/reference/api/eland.DataFrame.html). The eland's Data Frame wraps an OpenSearch index into a API similar to [pandas](https://pandas.pydata.org/), giving you the ability to process large amounts of data from OpenSearch inside a Jupyter Notebook.
```

Comment:
eland's should be replaced by opensearch-py-ml's

Reply:
@Naarcha-AWS , why close mark this as resolved ? I see "eland" not replaced by `opensearch-py-ml`